data/json/items/gun/ups.json

```diff
@@ -152,6 +152,20 @@
     "ammo_effects": [ "LASER", "ROBOT_DAZZLE" ],
     "flags": [ "NEVER_JAMS", "NO_UNLOAD", "NON_FOULING", "NEEDS_NO_LUBE" ]
   },
+ {
+    "id": "makeshift_scrambler",
+    "type": "GUN",
+    "copy-from": "hm12",
+    "name": { "str": "Makeshift Scrambler Rifle" },
+    "description": "This 'energy weapon' appears to be haphazardly jury-rigged from commercially available electronics. It has a radar dish placed where a barrel would be, and the frame of a weapon is covered in a mesh of wiring. Despite its appearance, this weapon can temporarily stun robots if you manage to hit their sensors.",
```

Comment:
```suggestion
    "description": "This 'energy weapon' appears to be haphazardly jury-rigged from commercially available electronics.  It has a radar dish placed where a barrel would be, and the frame of a weapon is covered in a mesh of wiring.  Despite its appearance, this weapon can temporarily stun robots if you manage to hit their sensors.",
```

Reply:
Oh, I see, will keep that in mind next time